src/Microsoft.ML.Transforms/TensorflowTransform.cs

```diff
@@ -0,0 +1,357 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.CommandLine;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Transforms;
+using Microsoft.ML.Transforms.TensorFlow;
+
+[assembly: LoadableClass(TensorflowTransform.Summary, typeof(IDataTransform), typeof(TensorflowTransform.Arguments), typeof(SignatureDataTransform),
+    TensorflowTransform.UserName, TensorflowTransform.ShortName)]
+
+// This is for de-serialization from a binary model file.
+[assembly: LoadableClass(typeof(TensorflowTransform.TensorFlowMapper), null, typeof(SignatureLoadRowMapper),
+    "", TensorflowTransform.TensorFlowMapper.LoaderSignature)]
+
+namespace Microsoft.ML.Transforms
+{
+    public static class TensorflowTransform
```

Comment:
Do we want to hide "TensorFlow" from the public API?  I thought the thinking was to hide the implementation details from the user.  Is that no longer a goal?

Reply:
Nevermind, I was thinking of pre-trained featurizers.

---
In reply to: [212470775](https://github.com/dotnet/machinelearning/pull/704#discussion_r212470775) [](ancestors = 212470775)